### DISCUSSION TITLE
antd5下修改主题色， FormDialog 主题色无法统一

### DISCUSSION THREAD

**Discussion by congtaoa** [alibaba/formily#4089] [Q&A 2.x]
antd5下修改主题色， FormDialog 主题色无法统一

**Comment by congtaoa** [comment]
使用<FormDialog.Portal> 包裹

**Reply by zxq08** [reply]
可以统一设置吗，我记得antd是可以全局的吧

**Reply by zXuff** [reply]
请问有其它统一设置的方法么，感觉用<FormDialog.Portal>包裹每个FormDrawer好麻烦
